src/app/services/wallet.service.ts

```diff
@@ -440,4 +443,8 @@ export class WalletService {
     this.intervalTime = (hasPendingTxs ? 20 : 60) * 1000;
     this.refreshBalancesTimeInSec = hasPendingTxs ? 20 : 300;
   }
+
+  private getCleanSeed(seed: string): string {
+    return seed.replace(/\r?\n|\r/g, ' ').replace(/ +/g, ' ').trim();
```

Comment:
Please change to `seed.replace(/(\n|\r\n)$/, '');`, as in the desktop wallet (https://github.com/skycoin/skycoin/blob/develop/src/gui/static/src/app/services/wallet.service.ts#L52). The goal is to only erase up to one new line character at the end of the string.